usr/local/www/diag_resetstate.php

```diff
@@ -65,73 +75,65 @@
 $pgtitle = array(gettext("Diagnostics"), gettext("Reset state"));
 include("head.inc");
 
+if ($input_errors)
+	print_input_errors($input_errors);
+
+if ($savemsg)
+	print('<div class="alert alert-success" role="alert">'.$savemsg.'</div>');
+
+$statetabelhelp =	'Resetting the state tables will remove all entries from the corresponding tables. This means that all open connections ' .
+					'will be broken and will have to be re-established. This may be necessary after making substantial changes to the ' .
+					'firewall and/or NAT rules, especially if there are IP protocol mappings (e.g. for PPTP or IPv6) with open connections.' .
+					'<br /><br />' .
+					'The firewall will normally leave the state tables intact when changing rules.' .
+					'<br /><br />' .
+					'<strong>NOTE:</strong> If you reset the firewall state table, the browser session may appear to be hung after clicking &quot;Reset&quot;. ' .
+					'Simply refresh the page to continue.';
+
+$sourcetablehelp =	'Resetting the source tracking table will remove all source/destination associations. ' .
+					'This means that the \"sticky\" source/destination association ' .
+					'will be cleared for all clients.' .
+					' <br /><br />' .
+					'This does not clear active connection states, only source tracking.';
+
+$tab_array = array();
+$tab_array[] = array(gettext("States"), false, "diag_dump_states.php");
+if (isset($config['system']['lb_use_sticky']))
```

Comment:
Indenting or accolades here should be improved. Is this if-statement intended to apply to the first, or both tabs?